paramiko/transport.py

```diff
@@ -17,6 +17,7 @@
 # along with Paramiko; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA.
 
+# flake8: noqa
```

Comment:
Yeah, I tried doing Flake8 for this module, and it was just overwhelming.  Hence the cop-out and I skipped checking the entire module.  A better approach would be to break out some of the deeply nested code into separate functions, but I wanted to concentrate on doing this in a separate PR.